docker/enrollment-server.xml

```diff
@@ -39,6 +39,14 @@
     <Parameter name="enrollment-server.inbox.enabled" value="${ENROLLMENT_SERVER_INBOX_ENABLED}"/>
     <Parameter name="enrollment-server.activation-spawn.enabled" value="${ENROLLMENT_SERVER_ACTIVATION_SPAWN_ENABLED}"/>
 
+    <!--  User-info configuration  -->
+    <Parameter name="enrollment-server.user-info.minimal-provider.enabled" value="${ENROLLMENT_SERVER_USER_INFO_MINIMAL_ENABLED}"/>
+    <Parameter name="enrollment-server.user-info.rest-provider.enabled" value="${ENROLLMENT_SERVER_USER_INFO_REST_ENABLED}"/>
+    <Parameter name="enrollment-server.user-info.rest-provider.restClientConfig.baseUrl" value="${ENROLLMENT_SERVER_USER_INFO_REST_URL}"/>
+    <Parameter name="enrollment-server.user-info.rest-provider.restClientConfig.httpDigestAuthEnabled" value="${ENROLLMENT_SERVER_USER_INFO_REST_DIGEST_ENABLED}"/>
```

Comment:
We want to use HTTP Basic authentication, because Spring only supports weak algorithms for HTTP Digest authentication. Please propagate to the whole pull request, thanks.

Reply:
Thanks for noticing. The client supports it, but yes, there is an issue on the server side you have mentioned.